src/db/migration/V1_1704373144572__Shallow_delete_hoks.sql

```diff
@@ -0,0 +1,667 @@
+CREATE OR REPLACE FUNCTION cascading_shallow_delete() RETURNS TRIGGER AS $$
+  DECLARE
```

Comment:
Note to self: katso propagoituuko myös osien poisto (ohi HOKSin, tai HOKSia päivitettäessä).